instance creation
with: anException
	^self new 
		signaledFailure: anException